randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -1680,6 +1687,16 @@ Extra - asset_id: collision_camera_015
               Speed Booster or Wall Jump (Beginner) or Simple IBJ
   > Door to White EMMI Arena (Bottom)
       Trivial
+  > Door to White EMMI Arena (Top)
+      Any of the following:
+          # https://youtu.be/RUvZaIl5vts
+          Diagonal Bomb Jump (Advanced) and Lay Normal Bomb
+          # https://youtu.be/otd7WX_dU_s
+          Speed Booster and Infinite Bomb Jump (Intermediate) and Speed Booster Conservation (Advanced) and Lay Normal Bomb
+          # https://youtu.be/OFHELQQbO4w
+          Speed Booster and Infinite Bomb Jump (Beginner) and Speed Booster Conservation (Intermediate) and Lay Cross Bomb
+          # https://youtu.be/vlhzedSfhEA
+          Morph Ball and Speed Booster and Speed Booster Conservation (Beginner) and Wall Jump (Beginner) and Use Spin Boost
```

Comment:
This shouldn't need the wall jump trick. I'm sure it could be done from the bottom door too and not need Morph.

Reply:
Or you can use upgrades.

Reply:
Also, we already have a Wall Jump requirement on jumping up to the Power door from here, so it seems consistent to also have it on this one. Though, it might make even more sense to add it to the bomb jump versions, unless we consider these to be feasible without a wall jump?